src/MinimalDelegation.sol

```diff
@@ -182,6 +183,12 @@ contract MinimalDelegation is
         if (isExpired) revert IKeyManagement.KeyExpired(expiry);
     }
 
+    /// @notice Returns true if the msg.sender is the executor or if the executor is address(0)
+    /// @param executor The address of the allowed executor of the signed batched call
+    function _senderCanExecute(address executor) private view returns (bool) {
```

Comment:
nit: maybe consider naming "senderIsExecutor" since can execute is related to spending guard related terminology?